src/test/java/com/google/cloud/tools/appengine/cloudsdk/PathResolverTest.java

```diff
@@ -17,6 +17,7 @@
 package com.google.cloud.tools.appengine.cloudsdk;
 
 import com.google.cloud.tools.test.utils.LogStoringHandler;
+import org.apache.commons.lang3.SystemUtils;
```

Comment:
There must be a way to check this without pulling in this dependency. Maybe 

System.getProperty("os.name").startsWith("Mac")